wandb/sdk/wandb_init.py

```diff
@@ -190,20 +210,24 @@ def setup(self, kwargs) -> None:  # noqa: C901
         )
 
         # merge config with sweep or sagemaker (or config file)
-        self.sweep_config = self._wl._sweep_config or dict()
+        self.sweep_config = dict()
+        sweep_config = self._wl._sweep_config or dict()
```

Comment:
Should this be `self.sweep_config`?

Reply:
No we need to split out the artifacts before we popultate the self.sweep_config